packages/api-explorer/package.json

```diff
@@ -9,6 +9,9 @@
   "license": "MIT",
   "author": "Looker",
   "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
Darn. Sorry I forgot about that when reviewing your other PR. Kinda silly to have two different toggles for it, IMO